src/components/MovieTile/MovieRating/index.js

```diff
@@ -9,11 +9,11 @@ export const MovieRating = ({ big, rating, votes }) => (
   <>
     {rating ? (
       <>
-        <MovieTileRating>
+        <MovieTileRating big={big}>
```

Comment:
I think you don't even need to use this syntax - big={big}. Just "big" as a parameter should be enough (not sure, but i've tried it on my branch and it works)

Reply:
You are probably right. Started with just "big" and don't even remember why I changed it. Thank you